### PR TITLE
Guillaume Savy - Carousel on desktop for review

### PR DESCRIPTION
**Pull request by guillsav**
# Description

Carousel done on Desktop view.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
Tested in development environment for desktop and mobile view.

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: landing/components/carousel/carousel.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    if (this.index >= this.storiesRefs.stories.length) {
+      this.index = 0;
+    }
+    TweenMax.from(this.storiesRefs.stories, 0.5, {x: '50px', opacity: '0'});
+    TweenMax.to(this.storiesRefs.stories, 0.5, {x: '0px', opacity: '1'});
+
+    this.storiesRefs.stories.forEach(story => {
+      story.style.display = 'none';
+    });
+    this.storiesRefs.stories[this.index].style.display = 'flex';
```
Comment (kamry-bowman) [inline landing/components/carousel/carousel.js]: a good pattern can be to have a class for these styles, and just toggle the class using `classList`, but this works too

**Review by kamry-bowman — DISMISSED** [review]
Great job, this looks awesome! Only note is to consider making the buttons on the carousel shrink slightly when you click them. I think that's a nice effect.. But  that's up to you!

**Comment by guillsav** [comment]
Actually that's a great idea to animate the buttons I will do that. I will also refactor the code and use classList that sounds better too. Thanks

**Review by kamry-bowman — APPROVED** [review]
Great work!

**Pull request merged**